app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe | str) -> None | str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            Cafe(cafe).visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy > 0:
+        return f"Friends should buy {masks_to_buy} masks"
+    return f"Friends can go to {cafe.name}"
```

Comment:
Bug: Here you are trying to access the `name` attribute of `cafe`, which is a string or a `Cafe` object. This will raise an `AttributeError` when `cafe` is a string. You should access the `name` attribute of the `Cafe` instance that you created.